components/script/script_thread.rs

```diff
@@ -2453,10 +2724,14 @@ impl ScriptThread {
         }
     }
 
-    fn handle_resize(&self, id: PipelineId, size: WindowSizeData, size_type: WindowSizeType) {
+    /// Batch windo resize operations into a single "update the rendering" task,
+    /// or, if a load is in progress,
+    /// set the window size directly.
```

Comment:
Nit: The linebreak is a bit odd here and there's a typo "windo".

```suggestion
    /// Batch window resize operations into a single "update the rendering" task,
    /// or, if a load is in progress, set the window size directly.
```